src/sentry/conf/server.py

```diff
@@ -1259,3 +1259,7 @@ def get_raven_config():
 }
 
 SOUTH_TESTS_MIGRATE = os.environ.get('SOUTH_TESTS_MIGRATE', '0') == '1'
+
+PUBSUB_ENABLED = False
```

Comment:
I would remove these out of here. We don't need to clutter actual conf for something effectively temporary.